src/pages/Order.jsx

```diff
@@ -0,0 +1,43 @@
+import GlobalTemplate from "@/components/templates/global-template/GlobalTemplate.jsx";
+import styled from "styled-components";
+import OrderDelivery from "@/components/organisms/order-delivery/OrderDelivery.jsx";
+import useGetCartItemsQuery from "@/hooks/useGetCartItemsQuery.js";
+import OrderProductColumn from "@/components/organisms/order-product-column/OrderProductColumn.jsx";
+import OrderAgreeTerm from "@/components/organisms/order-agree-term/OrderAgreeTerm.jsx";
+
+const Styled = {
+  Container: styled.div`
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  `,
+};
+function Order() {
+  const { data } = useGetCartItemsQuery();
```

Comment:
```suggestion
  const { data } = useGetCartItemsQuery();
  const { products = [] } = data ?? {};
```

와 같이 쓰면 옵셔널 체이닝을 조금 덜 쓰고 타입 안정을 보장할 수도 있어요!